apps/ui/src/components/MessageVotingPower.vue

```diff
@@ -0,0 +1,47 @@
+<script setup lang="ts">
+const props = defineProps<{
+  minVotingPower?: bigint;
+  minProposalThreshold?: bigint;
+  votingPower: {
+    status: string;
+    totalVotingPower: bigint;
+  };
+}>();
+
+defineEmits<{
+  (e: 'fetchVotingPower');
+}>();
+
+const insufficientVoteVp = computed(
+  () =>
+    props.votingPower.status === 'success' &&
+    props.minVotingPower !== undefined &&
+    props.votingPower.totalVotingPower <= props.minVotingPower
+);
+
+const insufficientProposeVp = computed(
+  () =>
+    props.votingPower.status === 'success' &&
+    props.minProposalThreshold !== undefined &&
+    props.votingPower.totalVotingPower < props.minProposalThreshold
+);
+</script>
+
+<template>
+  <div
+    v-if="votingPower.status === 'error'"
+    class="flex flex-col gap-3 items-start"
+    v-bind="$attrs"
+  >
+    <UiAlert type="error">There was an error fetching your voting power.</UiAlert>
+    <UiButton type="button" class="flex items-center gap-2" @click="$emit('fetchVotingPower')">
+      <IH-refresh />Retry
+    </UiButton>
+  </div>
+  <UiAlert v-else-if="insufficientVoteVp" type="error">
+    You do not have enough voting power to vote on this proposal.
```

Comment:
I would make it simpler just so it fit in 1 line.
```suggestion
    You do not have enough voting power to vote.
```